engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/dto/Internals.java

```diff
@@ -24,11 +24,14 @@
 public class Internals {
 
   public static final String SERIALIZED_APPLICATION_SERVER = "application-server";
+  public static final String SERIALIZED_CAMUNDA_INTEGRATION = "camunda-integration";
 
   protected Database database;
   @SerializedName(value = SERIALIZED_APPLICATION_SERVER)
   protected ApplicationServer applicationServer;
   protected Map<String, Command> commands;
+  @SerializedName(value = SERIALIZED_CAMUNDA_INTEGRATION)
+  protected Map<String, Object> camundaIntegration;
```

Comment:
Just an understanding question: do we need a map here, would a list/set of Strings be sufficient? we are not adding entries with `false` here, are we? Being in the list already means it part, right?